lib/expectations_runner.rb

```diff
@@ -38,10 +42,16 @@ def run_expectation!(expectation, ast)
   end
 
   def generate_ast!(source_code)
-    %x"#{gobstones_command} #{write_tempfile(source_code, 'gbs').path} --print-ast --target parse --no-print-retvals"
+    remove_compilation_steps %x"#{gobstones_command} #{write_tempfile(source_code, 'gbs').path} --print-ast --target parse --no-print-retvals"
   end
 
   def gobstones_command
     StonesSpec::Language::Gobstones.gobstones_command
   end
+
+  private
+
+  def remove_compilation_steps(output)
+    output.lines.drop(2).join
```

Comment:
In the `ErrorMessageParser`, but the number is 4 in that case (because 2 more steps are executed). What do you suggest?